config/locales/admin_center/en.yml

```diff
@@ -0,0 +1,14 @@
+---
+en:
+  admin_center:
+    actions:
+      toggle_on: 'ON'
```

Comment:
Is there a reason this attribute has quotes while the others do not?

Reply:
Yep, good question.  "On" is a reserved word in yml (it's treated as a boolean), so it's gotta be escaped.  See near the bottom of this section: https://guides.rubyonrails.org/i18n.html#providing-translations-for-internationalized-strings